server/src/inmantals/server.py

```diff
@@ -67,7 +67,7 @@ async def initialize(self, rootPath, rootUri, **kwargs):  # noqa: N803
         os.chdir(rootPath)
         init_options = kwargs.get("initializationOptions", None)
         if init_options:
-            self.compiler_venv_path = init_options.get("compilerVenv", None)
+            self.compiler_venv_path = init_options.get("compilerVenv", ".env-ls-compiler")
```

Comment:
The venv path is relative to the current working directory, not to the project root: `os.path.join(self.rootPath, ".env-ls-compiler")`